training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+
+## Pre requisites
+
+You need to complete the [set up your Mac for development work][setup-ons-mac]. If this has not been completed, please click on the link and follow the instructions. Afterwards, return back to this page.
+
+Run `dp remote allow develop` and `dp remote allow concourse` to ensure you can access concourse and the develop environment.
+Please note that if you have production access, you can run `dp remote allow production` to access production environment.
+
+[Cloud Technologies and Infrastructure](../services/INFRASTRUCTURE.md) - You should have an understanding of AWS and the infrastructure as the platform services are very closely related to this
+
+## Structure
+![](images/platform-structure.png)
+
+## Contents
+ - [Terraform](#terraform)
+ - [Ansible](#ansible)
+ - [Concourse](#concourse)
+ - [Consul](#consul)
+ - [Nomad](#nomad)
+ - [Kibana](#kibana)
+ - [Vault](#vault)
+ - [Grafana](#grafana)
+ - [Prometheus](#prometheus)
+
+## Terraform
+> Terraform is an open-source infrastructure as code software tool that provides a consistent CLI workflow to manage hundreds of cloud services. Terraform codifies cloud APIs into declarative configuration files. It allows infrastructure to be expressed as code in a simple, human readable language called HCL (HashiCorp Configuration Language). It reads configuration files and provides an execution plan of changes, which can be reviewed for safety and then applied and provisioned.
+
+Terraform provisioning builds the infrastructure for the test and live environments (`develop` and `production`) which involves setting up stacks (`app-users`, `web`, `publishing`, `mongodb`, etc). Therefore, whenever an infrastructure change is required, terraform needs to be run. The use of terraform is located in [dp-setup/terraform][dp-setup-terraform]
+ 
+More information about Terraform can be found on their website - https://www.terraform.io/
+
+## Ansible
+> Ansible is an open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code. It uses no agents and no additional custom security infrastructure, so it's easy to deploy and uses YAML, in the form of Ansible Playbooks, that allows to describe the automation jobs. 
+
+Ansible provisioning can be considered as adding and configuring applications *to* the test and production service environments in the cloud (`develop` and `production`). Therefore, whenever the change of the infrastructure *of the app* is required, ansible needs to be run. The use of ansible is located in [dp-setup/ansible][dp-setup-ansible]
+
+More information about Ansible can be found on their website - https://www.ansible.com/overview/how-ansible-works
+
+## Concourse
```

Comment:
There might be a general question we need to answer about these docs/a mindset I have that I need to change - around whether we link to private repos from these public docs. We have historically tried to minimize that, but I think where you've done it above for ansible and terraform it adds a lot of value. 

I think linking to the code for how we manage concourse at some point in this section might be worthwhile, but we might just want to check when people are back that theres concensus that providing the value to our team is more important than the bad experience to external people if they can't follow all the links